openfe/setup/transformation_factory/easy_transformation_factory.py

```diff
@@ -0,0 +1,81 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+from typing import FrozenSet, Dict
+
+from gufe import (
+    AlchemicalNetwork,
+    Protocol,
+    LigandAtomMapping,
+    Transformation,
+    SmallMoleculeComponent,
+)
+from gufe import Transformation
+
+from .abstract_transformation_factory import AbstractTransformerFactory
+from ..chemicalsystem_generator.abstract_chemicalsystem_generator import (
+    AbstractChemicalSystemGenerator,
+)
+
+
+class AbstractEasyTransformationFactory(AbstractTransformerFactory):
+    def __init__(
+        self,
+        protocol: Protocol,
+        chemical_system_generator: AbstractChemicalSystemGenerator,
+    ):
+        self.protocol = protocol
+        self.chemical_system_generator = chemical_system_generator
+
+
+class RFETransformationFactory(AbstractEasyTransformationFactory):
+    """
+    Simple transformer is a very simplistic transformer, that mapps a defined protocol to a given set of transformations
+    """
+
+    def __call__(
+        self, alchemical_network_edges: FrozenSet[LigandAtomMapping], name="approach"
+    ) -> AlchemicalNetwork:
+        return self._build_alchemical_network(
+            alchemical_network_edges=alchemical_network_edges, name=name
+        )
+
+    def _build_alchemical_network(
+        self, alchemical_network_edges: FrozenSet[LigandAtomMapping], name="approach"
+    ) -> AlchemicalNetwork:
+        name = name
+        edges = []
+        nodes = []
+
+        for edge in alchemical_network_edges:
+
+            for stateA_env, stateB_env in zip(
+                self.chemical_system_generator(edge.componentA),
+                self.chemical_system_generator(edge.componentB),
+            ):
+                transformation_name = (
+                    name + "_" + stateA_env.name + "_" + stateB_env.name
+                )
+
+                #Todo: Another dirty hack!
+                if("vacuum" in transformation_name):
+                    protocol_settings = self.protocol._default_settings()
+                    protocol_settings.system_settings.nonbonded_method = "nocutoff"
```

Comment:
push to protocol!

Reply:
Context here? It looks like you're saying you want to by default push changing the nonbonded method setting to something else if it's a vacuum transformation - that would definitely be a no.

Reply:
was solved during discussions yesterday.